project/MicroServiceBuild.scala

```diff
@@ -9,13 +9,14 @@ object MicroServiceBuild extends Build with MicroService {
   override lazy val appDependencies: Seq[ModuleID] = compile ++ test()
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "play-reactivemongo" % "6.0.0",
+    "uk.gov.hmrc" %% "play-reactivemongo" % "5.2.0",
```

Comment:
downgrade?

Reply:
Didn't work first-time with the new version, so I reverted and then forgot. Can revisit...